src/components/form/index.tsx

```diff
@@ -0,0 +1,148 @@
+import PopUp from '#components/pop-up'
+import styled from '@emotion/styled'
+import React from 'react'
+
+function Feedback() {
+  const [isDisplayed, setDisplay] = React.useState(false)
+
+  function handleSubmit(e: React.FormEvent<HTMLFormElement>): void {
+    e.preventDefault()
+    const {
+      name,
+      raidName,
+      rate,
+      description,
+    } = e.target as typeof e.target & {
+      name: { value: string }
+      raidName: { value: string }
+      rate: { value: string }
+      description: { value: string }
+    }
+    console.log({
+      name: name.value,
+      raidName: raidName.value,
+      rate: rate.value,
+      description: description.value,
+    })
```

Comment:
do we need this? 🤔

Reply:
because I dont know where the data will go

Reply:
then remove the `console.log` please, maybe place a TODO comment there or something

Reply:
sure i can remove the console.log and leave the object commented out with TODO in the start